app/packages/backend/src/plugins/scaffolder.ts

```diff
@@ -14,15 +14,21 @@ import {
   createGithubTeamAction,
   addGithubTeamToRepoAction,
   filters,
+  addDeliveryProjectToRepo,
+  createGithubClient,
 } from '@internal/backstage-plugin-scaffolder-backend-module-adp-scaffolder-actions';
 import { createHttpBackstageAction } from '@roadiehq/scaffolder-backend-module-http-request';
+import { AdpClient } from '@internal/plugin-adp-backend';
 
 export default async function createPlugin(
   env: PluginEnvironment,
 ): Promise<Router> {
   const catalogClient = new CatalogClient({
     discoveryApi: env.discovery,
   });
+  const adpClient = new AdpClient({
```

Comment:
CatalogClient = client for the endpoints exposed by the catalog plugin, so AdpClient = client for the endpoints exposed by the adp plugin. I have only moved the endpoints I use in the PR to this class so far, but going forwards I think we should move all our adp api calls to go through this